### DISCUSSION TITLE
Quiz_02-002

### DISCUSSION THREAD

**Discussion by edoyle724** [cooperrc/engineering-dynamics#10] [Q&A]
When it asks for constraint equations and what the vector equation is, which example from the homework is this most similar to? I'm having trouble remembering how to set up constraint equations and vector equations and I'm assuming this is based off one of the homework problems. I'm guessing it's similar to the four-bar linkage example, but I'm not sure which equations are what. Thanks!

**Comment by cooperrc** [comment]
There's an example in the readings too:
https://cooperrc.github.io/engineering-dynamics/module_02/mechanical-advantage.html#kinematics-geometry-of-motion
